src/ZkFold/Cardano/OffChain/E2E.hs

```diff
@@ -32,12 +33,14 @@ data IdentityCircuitContract = IdentityCircuitContract {
 } deriving stock (Show, Generic)
   deriving anyclass (ToJSON, FromJSON)
 
-deriving anyclass instance FromJSON (V.Vector 19 Fr)
+deriving newtype instance FromJSON (V.Vector 19 Fr)
 deriving anyclass instance ToJSON   (PlonkupProverSecret BLS12_381_G1)
 deriving anyclass instance FromJSON (PlonkupProverSecret BLS12_381_G1)
```

Comment:
I think we need to make a pr in `zkfold-base` and remove all `PlonkupProverSecret` `deriving` staff.

Reply:
Will let @vlasin decide if we make an new issue for this.